### PR TITLE
HT-2475 Load export file into holdings_htitem_htmember

### PR DESCRIPTION
**Pull request by billdueber**
HT-2475 Load export file into holdings_htitem_htmember

Created `bin/load_overlap_report_to_database.rb` to take the out from `bin/export_overlap_report.rb`
and import it into mysql table holdings_htitem_htmember.

The import process will automatically split up an import file into temporary partial files and then slurp
the data into mysql via LOAD DATA INFILE, pausing between sub-files to allow mysql instances to sync up.

Notable changes:
  * Addition of holdings_htitem_htmember sql definition (with new cluster_id column)
  * `HoldingsDB.new` now returns a new object (which mostly just delegates everything to the underlying Sequel connection)
  * Added `HoldingsDB#load_tab_delimited_file` to do the whole split/load/pause cycle described above
  * Set a default logger (targeting STDERR) in `Services.logger`

Things to revisit:
  * Keys on the target table should probably be dropped before the import and then re-created, but I'd like to
    build real infrastructure for that (do we need migrations???)

### CODE REVIEW

**Comment by aelkiss** [comment]
Looks good to me. In particular, I think this will be fairly straightforward to extend to do delta processing -- there would be an additional step after `export_overlap_report` to compute the delta; then, `load_tab_delimited_file` can process the additions, and we can implement a new method to process the deletions.

**Comment by billdueber** [comment]
I'm confused:

```
diff lib/autoscrub.rb out/production/holdings-backend/autoscrub.rb #=> no diffs
rubocop out/production/holdings-backend/autoscrub.rb # 2 warnings
rubocop lib/autoscrub.rb # 0 warnings
```

File: lib/services.rb [hunk 1]
```diff
@@ -2,11 +2,14 @@
...
+require "logger"
 
 Services = Canister.new
 
-Services.register(:holdings_db) { HoldingsDB.connection }
-Services.register(:ht_members) { HTMembers.new }
+Services.register(:holdings_db) { HoldingsDB.new }
 Services.register(:ht_collections) { HTCollections.new }
+Services.register(:ht_members) { HTMembers.new }
+Services.register(:logger) { Logger.new(STDERR) }
```
[the remark this answers is not in the document]
Comment (aelkiss) [reply lib/services.rb:14]: Sounds good.

**Comment by aelkiss** [comment]
The rubocop complaints are:

```bin/load_overlap_report_to_database.rb:56:72: C: Layout/TrailingEmptyLines: Final newline missing.

lib/services.rb:15:1: C: Layout/TrailingEmptyLines: 1 trailing blank lines detected.```

Possible you're running a different rubocop than is in the bundle?

**Pull request merged**